webodm.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
```

Comment:
`set -eo pipefail` (`-o pipefail` is a bash-ism) is nice; makes things fail early and often (w/o `pipefail` errors sometime get masked).